internal/server/patch/data.go

```diff
@@ -274,7 +275,7 @@ func (s *DataServer) sendFileHeader(c *server.Client, patch *fileEntry) error {
 func (s *DataServer) sendFileChunk(c *server.Client, chunk, chksm, chunkSize uint32, fdata []byte) error {
 	if chunkSize > maxFileChunkSize {
 		archon.Log.Errorf("Attempted to send %v byte chunk; max is %v",
-			string(chunkSize), string(maxFileChunkSize))
+			strconv.Itoa(int(chunkSize)), string(rune(maxFileChunkSize)))
```

Comment:
Go wouldn't build with these being casted to string - I think it's a Go 1.16 thing.